src/main/java/org/sagebionetworks/web/client/widget/profile/UserProfileEditorWidgetViewImpl.java

```diff
@@ -388,13 +400,14 @@ public void setSynAlert(IsWidget w) {
 
   @Override
   public void setOwnerId(String userId) {
-    ReactNode accountLevelBadgeComponent =
+    ReactNode accountLevelBadgesComponent =
       React.createElementWithSynapseContext(
-        SRC.SynapseComponents.AccountLevelBadge,
-        AccountLevelBadgeProps.create(userId),
+        SRC.SynapseComponents.AccountLevelBadges,
+        AccountLevelBadgesProps.create(userId),
         propsProvider.getJsInteropContextProps()
       );
-    accountLevelBadgeContainer.render(accountLevelBadgeComponent);
+    accountLevelBadgesContainer.render(accountLevelBadgesComponent);
+    setAccountTypeVisibility(Long.parseLong(userId));
```

Comment:
When the user doesn't have any badges (i.e. not certified nor validated), the Account Type row should be hidden